x-pack/legacy/plugins/apm/server/lib/transactions/breakdown/index.ts

```diff
@@ -24,6 +30,47 @@ export async function getTransactionBreakdown({
 }) {
   const { uiFiltersES, client, config, start, end } = setup;
 
+  const subAggs = {
+    sum_all_self_times: {
+      sum: {
+        field: 'span.self_time.sum.us'
+      }
+    },
+    total_transaction_breakdown_count: {
+      sum: {
+        field: 'transaction.breakdown.count'
+      }
+    },
+    types: {
+      terms: {
+        field: 'span.type',
+        size: 20,
+        order: {
+          _count: 'desc'
+        }
+      },
+      aggs: {
+        subtypes: {
+          terms: {
+            field: 'span.subtype',
```

Comment:
many of these fields are defined as constants in `common/elasticsearch_fieldnames.ts`

Reply:
changed all of the field names to constants from `common/elasticsearch_fieldnames.ts`